excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ParseContext.java

```diff
@@ -241,14 +234,14 @@ public CaseFieldEntity getCaseFieldForCaseType(String caseTypeId, String caseFie
         return Optional.ofNullable(baseTypes.get(reference));
     }
 
-    public Optional<UserRoleEntity> getIdamRole(final String role) {
-        return Optional.ofNullable(userRoles.get(role));
+    public Optional<AccessProfileEntity> getIdamAccessProfile(final String accessProfile) {
+        return Optional.ofNullable(accessProfiles.get(accessProfile));
     }
 
-    public Optional<UserRoleEntity> getRole(String caseType, final String role) {
-        Optional<UserRoleEntity> userRoleEntity = Optional.ofNullable(userRoles.get(role));
-        if (userRoleEntity.isPresent()) {
-            return userRoleEntity;
+    public Optional<AccessProfileEntity> getAccessProfile(String caseType, final String role) {
```

Comment:
rename param

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f